_pages/dev/seo/structured-data.md

```diff
@@ -103,4 +96,6 @@ The given json schema will generate the following script tag:
 </script>
 ```
 
-**Note:** Spartacus doesn't validate the given JSON-LD schema.
+**Note:** Spartacus does not validate the given JSON-LD schema.
+
+<!-- Is this note saying that Spartacus does not validate any JSON-LD schemas, or just the one in the example provided here? -->
```

Comment:
Non of them. The given schema is written inside the script tag.

Reply:
Note updated to say:

> Spartacus does not validate the JSON-LD schema that is generated inside the script tag.